### DISCUSSION TITLE
Is it possible to use icon from font awesome for edit and delete button

### DISCUSSION THREAD

**Discussion by jonathanoeijoeng** [rappasoft/laravel-livewire-tables#843] [Q&A]
Hi,

Is it possible to use icon from fontawesome for edit and delete button as below?

<img width="825" alt="Screen Shot 2022-07-12 at 11 44 35" src="https://user-images.githubusercontent.com/7046560/178410482-8a310050-7424-457b-9aa2-17a074a54931.png">

**Comment by elsoftpy** [comment]
I tried publishing the views files, then changed the file

resources/views/vendor/livewire-tables/includes/columns/link.blade.php

instead escaping the output with {{ }} I tried printing the `$title` with {!! !!} and it worked

Original view
`<a href="{{ $path }}" {!! count($attributes) ? $column->arrayToAttributes($attributes) : '' !!}>{{ $title }}</a>`

Custom view
`<a href="{{ $path }}" {!! count($attributes) ? $column->arrayToAttributes($attributes) : '' !!}>{!! $title !!}</a>`

The LinkColumn looks like this
`LinkColumn::make('Edit')->title(fn($row) => '<i class="fa fa-pen"></i>')->location(fn($row) => route('edit-route', $row))`
